frontend/components/SetupPage/index.tsx

```diff
@@ -98,8 +99,18 @@ export const Setup = () => {
   }, [setupObject.state]);
 
   if (screenWithoutCards.includes(setupObject.state)) {
-    return setupScreen;
+    return (
+      <>
+        <SupportButton />
+        {setupScreen}
+      </>
+    );
   }
 
-  return <SetupCard>{setupScreen}</SetupCard>;
+  return (
+    <>
+      <SupportButton />
+      <SetupCard>{setupScreen}</SetupCard>
+    </>
+  );
 };
```

Comment:
tiny, but maybe we could have a Wrapper not to repeat the SupportButton twive... smth like
```
let Wrapper: React.ElementType = SetupCard;
if (screenWithoutCards.includes(setupObject.state)) {
  Wrapper = React.Fragment;
}

return (
  <Wrapper>
    <SupportButton />
    {setupScreen}
  </Wrapper>
);
  ```